app/models/talking.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class Talking < ApplicationRecord
```

Comment:
Try to add the model name of a resource instead an action; actions should be the name of functions or commands.

A proper name here would be `Talk` or Event

Reply:
on it